static/app/utils/profiling/guards/profile.tsx

```diff
@@ -30,3 +30,21 @@ export function isSentrySampledProfile(
     'frames' in profile.profile
   );
 }
+
+export function isSentryContinuousProfile(
+  profile: any
+): profile is Profiling.ContinuousProfile {
+  return (
+    'samples' in profile &&
+    'stacks' in profile &&
+    'frames' in profile &&
+    !('type' in profile) &&
+    !Array.isArray(profile.resources)
+  );
```

Comment:
Won't this also return true for the sentry sampled profile?

Reply:
I think it wont, but this is brittle code as the formats are not very consistent (future changes might break this). Let me explicitly force it down this code path